OpenUtau.Core/Classic/ExeResampler.cs

```diff
@@ -34,6 +36,27 @@ internal class ExeResampler : IResampler {
             }
         }
 
+        void FixMoreConfig(string moreConfigPath) {
+            var lines = new List<string> { };
+            if (File.Exists(moreConfigPath)) {
+                lines = File.ReadAllLines(moreConfigPath).ToList();
+            }
+            for (int i = 0; i < lines.Count; i++) {
+                if (lines[i].StartsWith("resampler-compatibility")) {
+                    if(lines[i] == "resampler-compatibility on"){
+                        //moreconfig.txt is correct
+                        return;
+                    } else {
+                        lines[i] = "resampler-compatibility on";
+                        File.WriteAllLines(moreConfigPath, lines);
+                        return;
```

Comment:
Does this break the moresampler as both resampler and wavtool use case?

Reply:
even the option `resampler-compatibility` is on, both moresampler as a wavtool and resampler still functions normally